src/Library/demos/Launcher/main.blp

```diff
@@ -23,73 +23,72 @@ Adw.StatusPage {
         Gtk.StackPage {
           name: "file_launcher";
           title: _("File Launcher");
-          child:
+          child: Box {
+            orientation: vertical;
+            halign: center;
+            spacing: 24;
             Box {
-              orientation: vertical;
               halign: center;
-              spacing: 24;
-              Box {
-                halign: center;
-                spacing: 18;
-                orientation: horizontal;
+              spacing: 6;
+              orientation: horizontal;
+              homogeneous: true;
 
-                Button launch_file {
-                  label: _("Launch");
-                  styles ["suggested-action", "pill"]
-                }
-                Button file_location {
-                  label: _("View");
-                  styles ["suggested-action", "pill"]
-                }
+              Button launch_file {
+                label: _("Launch");
+                styles ["suggested-action", "pill"]
               }
-
-              Button change_file {
-                label: _("Select File");
-                child: Box {
-                  halign: center;
-                  orientation: vertical;
-                  Label {
-                    label: _("Select File");
-                    styles ["title-4"]
-                  }
-                  Label file_name {
-                    label: _("workbench.txt");
-                    styles ["dim-label", "body"]
-                  }
-                };
-              }
-              LinkButton {
-                label: _("API Reference");
-                uri : "https://docs.gtk.org/gtk4/class.FileLauncher.html";
+              Button file_location {
+                label: _("View in Files");
```

Comment:
renamed from "View" to "View in Files" which is the recommended label for this

Reply:
Sorry, where is this recommendation found?

Reply:
I asked Tobias a while ago :p